[CF4SaaS] Various updates (#8935)
cloudflare/cloudflare-docs: content/cloudflare-for-platforms/cloudflare-for-saas/domain-support/hostname-validation/_index.md

---
pcx_content_type: concept

> title: Validate hostnames

Reviewer: Since this is a Concept page, should the title be a noun (e.g. "Hostname validation")?

> title: Validate hostnames

Author: ```suggestion
title: Hostname validation
```

weight: 2
---

> # Validate hostnames

Author: ```suggestion
# Hostname validation
```

Before Cloudflare can proxy traffic through a custom hostname, we need to verify your customer's ownership of that hostname.

{{<Aside type="note">}}

If a custom hostname is already on Cloudflare, using the [pre-validation methods](/cloudflare-for-platforms/cloudflare-for-saas/domain-support/hostname-validation/pre-validation/) will not shift the traffic to the SaaS zone. That will only happen once the [DNS target](/cloudflare-for-platforms/cloudflare-for-saas/start/getting-started/#step-3--have-customer-create-cname-record) of the custom hostnames changes to point to the SaaS zone.

{{</Aside>}}

## Options

> If minimizing downtime is more important to you, check out our [pre-validation methods](/cloudflare-for-platforms/cloudflare-for-saas/domain-support/hostname-validation/pre-validation/).

Reviewer: ```suggestion
If minimizing downtime is more important to you, refer to our [pre-validation methods](/cloudflare-for-platforms/cloudflare-for-saas/domain-support/hostname-validation/pre-validation/).
```
